newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/nrql/NrqlCondition.java

```diff
@@ -0,0 +1,55 @@
+package com.ocadotechnology.newrelic.alertsconfigurator.configuration.condition.nrql;
```

Comment:
Shouldn't it be in `com.ocadotechnology.newrelic.alertsconfigurator.configuration.condition` package?